docs/security.rst

```diff
@@ -0,0 +1,68 @@
+Security
+========
+
+Security is very important to us at Read the Docs. We are committed to responsible reporting and disclosure of security issues.
+
+Reporting a security issue
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you believe you've discovered a security issue at Read the Docs, please contact us at **security@readthedocs.org** (optionally using our :ref:`pgp-key`). We request that you please not publicly disclose the issue until it has been addressed by us.
+
+You can expect:
+
+* We will respond acknowledging your email typically within one business day.
+* We will follow up if and when we have confirmed the issue with a timetable for the fix.
+* We will notify you when the issue is fixed.
+* We will add the issue to our :ref:`security issue archive <security-issue-archive>`.
+
+.. _pgp-key:
+
+PGP key
+~~~~~~~
+
+You may use this `PGP key`_ to securely communicate with us and to verify signed messages you receive from us.
+
+.. _PGP key: http://pgp.mit.edu:11371/pks/lookup?op=get&search=0x71337C3047A1B066
```

Comment:
I believe this is my PGP key, and I'm not confident I still have access to it. Perhaps we should generate a new one via keybase or something, perhaps that we can share with the team?

Reply:
Oh, if you don't have access to it that is definitely a problem. I'll create a new one and share the key.